modules/bitgo/example/ts/algo/transaction-with-emergency-param.ts

```diff
@@ -0,0 +1,83 @@
+import { BitGo, Wallet } from 'bitgo';
```

Comment:
Missing title comments. See other examples.

Reply:
Let's add `Copyright 2020, BitGo, Inc.  All Rights Reserved.` and we're good to go (for consistency).